BackEnd/src/business/games.ts

```diff
@@ -40,6 +41,7 @@ export async function SaveDataByMatchId(matchId: string, updatePlayerStats: bool
   const savedGameModel: GameModel = await SaveSingleMatchById(matchId, gameData);
 
   if (updatePlayerStats) {
+    await buildRisenTeams();
```

Comment:
Should this be moved to a higher level function? Thoughts?

Reply:
I dont think this should be in a higher level function because its directly related to game saving but im looking at this code again and i realize that i made a mistake. I think this call needs to happen before the games save.

Will update and test